data_submission/src/config.rs

```diff
@@ -225,13 +229,20 @@ impl AppConfig {
             index += 1;
         }
 
+        let enigma_encryption_service_url = env::var("ENCIPHER_ENCRYPTION_SERVICE_URL")?;
+        // default to v1 if not set
+        let enigma_encryption_service_version =
+            env::var("ENCIPHER_ENCRYPTION_SERVICE_VERSION").unwrap_or(String::from("v1"));
+
```

Comment:
incorrect now, will now work, .env expects another name anyway. Change to enigma